requirements/doc.txt

```diff
@@ -24,7 +24,7 @@ click==8.1.3
     #   code-annotations
 code-annotations==1.3.0
     # via -r requirements/test.txt
-coverage[toml]==6.4.4
+coverage[toml]==6.5.0
```

Comment:
No breaking change in the new version: https://github.com/nedbat/coveragepy/blob/master/CHANGES.rst#version-650--2022-09-29